### ISSUE TITLE
ActionView::Template::Error: Couldn't find Holder with 'id'=198

### ISSUE THREAD

**Issue by voodoorai2000** [AyuntamientoMadrid/agendas#311]
View details in Rollbar: [https://rollbar.com/consul/Agendas/items/38/](https://rollbar.com/consul/Agendas/items/38/)

```

ActiveRecord::RecordNotFound: Couldn't find Holder with 'id'=198
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activerecord-4.2.10/lib/active_record/core.rb", line 155, in find
  File "/aytomad/app/agendas/agendas/releases/20171230100311/app/views/visitors/headline/_holder.html.erb", line 4, in _app_views_visitors_headline__holder_html_erb__3538579874552263353_64354400
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/template.rb", line 145, in block in render
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/notifications.rb", line 164, in block in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/notifications/instrumenter.rb", line 20, in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/notifications.rb", line 164, in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/template.rb", line 333, in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/template.rb", line 143, in render
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/partial_renderer.rb", line 339, in render_partial
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/partial_renderer.rb", line 310, in block in render
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/abstract_renderer.rb", line 39, in block in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/notifications.rb", line 164, in block in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/notifications/instrumenter.rb", line 20, in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/notifications.rb", line 164, in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/abstract_renderer.rb", line 39, in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/partial_renderer.rb", line 309, in render
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/renderer.rb", line 51, in render_partial
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/helpers/rendering_helper.rb", line 35, in render
  File "/aytomad/app/agendas/agendas/releases/20171230100311/app/helpers/application_helper.rb", line 4, in show_headline
  File "/aytomad/app/agendas/agendas/releases/20171230100311/app/views/visitors/index.html.erb", line 10, in _app_views_visitors_index_html_erb___2668946671925517472_31532560
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/template.rb", line 145, in block in render
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/notifications.rb", line 164, in block in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/notifications/instrumenter.rb", line 20, in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/notifications.rb", line 164, in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/template.rb", line 333, in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/template.rb", line 143, in render
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/template_renderer.rb", line 54, in block (2 levels) in render_template
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/abstract_renderer.rb", line 39, in block in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/notifications.rb", line 164, in block in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/notifications/instrumenter.rb", line 20, in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/notifications.rb", line 164, in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/abstract_renderer.rb", line 39, in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/template_renderer.rb", line 53, in block in render_template
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/template_renderer.rb", line 61, in render_with_layout
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/template_renderer.rb", line 52, in render_template
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/template_renderer.rb", line 14, in render
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/renderer.rb", line 46, in render_template
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/renderer.rb", line 27, in render
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/rendering.rb", line 100, in _render_template
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/action_controller/metal/streaming.rb", line 217, in _render_template
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/rendering.rb", line 83, in render_to_body
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/action_controller/metal/rendering.rb", line 32, in render_to_body
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/action_controller/metal/renderers.rb", line 37, in render_to_body
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/abstract_controller/rendering.rb", line 25, in render
  File "/aytomad/app/agenda

**Comment by raul-fuentes** [comment]
I'll do this one

**Comment by Senen** [comment]
Fixed via #322
